fiberzerolog/config.go

```diff
@@ -79,6 +80,13 @@ type Config struct {
 	// Optional. Default: {"ip", "latency", "status", "method", "url", "error"}
 	Fields []string
 
+	// Wrap headers to dictionary.
+	// If false: {"method":"POST", "header-key":"header value"}
+	// If true: {"method":"POST", "reqHeaders": {"header-key":"header value"}}
+	//
+	// Optional. Default: false
+	WrapHeaders bool
```

Comment:
@roman-wb 
documentation is important, pls add some infos to the markdown file
add it to the table 
https://docs.gofiber.io/contrib/next/fiberzerolog/

Reply:
Added to this repo and Docs https://github.com/gofiber/docs/pull/388

Reply:
Thanks, only the adaptation of the readme in this repository is necessary
The docs repository is automatically synchronized on release